frontend/document/koptinterface.lua

```diff
@@ -289,6 +289,20 @@ function KoptInterface:getCoverPageImage(doc)
     end
 end
 
+function KoptInterface:getNativePageImage(doc , pageno)
+    --[[cancel zoom to get better image quality for zooming
+
+    local native_size = Document.getNativePageDimensions(doc, pageno)
+    local canvas_size = CanvasContext:getSize()
+    local zoom = math.min(canvas_size.w / native_size.w, canvas_size.h / native_size.h)
+    ]]
```

Comment:
Might replace that with a comment `-- Unlike in getCoverPageImage(), we cancel zoom to get better image quality for zooming` - although written like that, I don't understand it very well :/ "cancell zoom to get better zoom" ? :)

Reply:
sorry for englisg :)
corrected